character select screen tweak

nickrehac opened the pull request:
make character/hat selection more obvious by keeping eyes mostly in the center of the character panel

![Screenshot from 2024-02-03 17-02-54](https://github.com/fishfolk/jumpy/assets/158633087/e0c42db5-ebdf-4d8c-9d00-23379aa30c5a)
![Screenshot from 2024-02-03 17-02-47](https://github.com/fishfolk/jumpy/assets/158633087/d060c45a-11c9-4c18-8b35-6a9923f3459b)
![Screenshot from 2024-02-03 17-02-42](https://github.com/fishfolk/jumpy/assets/158633087/85dc055b-479c-4f2d-b71f-244f8ab561a8)

zicklag reviewed (changes requested):
Hey there, thanks for the PR!

One thing I would like to do is make sure that the player stays the same size before and after selecting the skin. For example, in the screenshot below, the player on the left that hasn't been locked in, is smaller than the skin after you confirm the player selection. 

![image](https://github.com/fishfolk/jumpy/assets/25393315/01ec4efb-1df7-4f3d-bd99-ebef5b28cc45)

We should try to keep the player the same size before and after confirming.

Another minor tweak is to use the pixel font and size for the hat selection name:

![image](https://github.com/fishfolk/jumpy/assets/25393315/f00d2fa5-af64-4064-b3e6-da8e877de5f9)

I would also like just a little bit more space between the hat's label, and the hat itself. Just a little. 🤏

nickrehac commented:
Thanks for the feedback!
I looked into the character scaling and found the sprite sheet has a considerable amount of padding which throws off the ui and might be better suited as part of the ui construction itself. I'm going to check if that padding is needed elsewhere

nickrehac commented:
fixed the size and added spacing.
characters drop down when selected which could be an indicator of selection status, but is actually egui being quirky
thoughts?

![Screenshot from 2024-02-06 13-57-11](https://github.com/fishfolk/jumpy/assets/158633087/9b1ec0b9-4258-42c9-9c5c-bfc410f8d3a5)

zicklag commented:
Hi @nickrehac, sorry for the delay. Life has had me incredibly busy recently. Could you push the code with the updated size and spacing?

As far as the egui issue, there must be some way to fix it, but me or someone else can try to take a look at it if we get the chance. I do want to get that fixed before merging.

nickrehac commented:
Here you go, and no worries.
I might take a crack at this last bit soon. It seems that calculating the sprite size each frame is unavoidable
